packages/instrumentation-redis/package.json

```diff
@@ -50,22 +50,15 @@
     "access": "public"
   },
   "peerDependencies": {
-    "@opentelemetry/api": "^1.3.0"
+    "@opentelemetry/api": "^1.3.0",
+    "redis": "^5.6.0"
```

Comment:
Are we sure we want a change in peer deps here?

Reply:
I'm not super familiar with how TAV works but when it installs different versions of redis while testing, the multiple versions were conflicting because of the presence of v5. I would like the owner to review if possible.